src/modm/platform/dma/stm32/module.lb

```diff
@@ -24,6 +24,8 @@ def prepare(module, options):
     # FIXME the driver is for L4 only
     if device.identifier["family"] not in ["l4"]:
         return False
+    if device.identifier["name"] not in ["31", "32", "43"]:
+        return False
```

Comment:
```suggestion
    if device.identifier["family"] not in and device.identifier["name"] not in ["31", "32", "43"]:
        return False
```

Reply:
I typically try to keep it simple and readable, but for you I'll do an exception.
😅

Reply:
`identifier["name"] not in ["31", "32", "43"]` makes only sense in this case in combination with `identifier["family"] not in ["l4"]` ...

Reply:
I reverted it, makes it simple. I expect to add more in the near future when DMA support will extend to other STM32.